baseplate/context/memcache/lib.py

```diff
@@ -0,0 +1,108 @@
+from six import string_types
```

Comment:
a module docstring would be nice too. since this is sorta hidden internal stuff we don't need to sphinx it up though.